quickstarts/microsoft.appconfiguration/app-configuration/main.bicep

```diff
@@ -0,0 +1,161 @@
+@description('Specify the locations for all resources.')
+param location string = resourceGroup().location
+
+@description('Specify the virtual machine admin user name.')
+param adminUsername string
+
+@description('Specify the virtual machine admin password.')
+@secure()
+param adminPassword string
+
+@description('Specify the DNS label for the virtual machine public IP address. It must be lowercase. It should match the following regular expression, or it will raise an error: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$.')
+param domainNameLabel string
+
+@description('Specify the size of the VM.')
+param vmSize string = 'Standard_D2_v3'
+
+@description('Specify the storage account name.')
+param storageAccountName string
+
+@description('Name of the resource group for the app config store.')
+param appConfigStoreResourceGroup string
+
+@description('App configuration store name.')
+param appConfigStoreName string
+
+@description('Specify the name of the key in the app config store for the VM windows sku.')
+param vmSkuKey string
+
+@description('Specify the name of the key in the app config store for the VM disk size')
+param diskSizeKey string
+
+var nicName = 'myVMNic'
+var addressPrefix = '10.0.0.0/16'
+var subnetName = 'Subnet'
+var subnetPrefix = '10.0.0.0/24'
+var publicIPAddressName = 'myPublicIP'
+var vmName = 'SimpleWinVM'
+var virtualNetworkName = 'MyVNET'
+var subnetRef = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, subnetName)
+var appConfigRef = resourceId(appConfigStoreResourceGroup, 'Microsoft.AppConfiguration/configurationStores', appConfigStoreName)
```

Comment:
create a symbolic name for this using the ```existing``` syntax

Reply:
Resolved in latest commit